src/helpers.ts

```diff
@@ -91,3 +91,34 @@ export function isMac() {
     navigator?.userAgentData?.platform || navigator?.platform || 'unknown'
   return platform.toUpperCase().indexOf('MAC') !== -1
 }
+
+/**
+ * Sort ordening groups
+ *
+ * @return Array
+ */
+export function sortGroupOrder(
+  a: string,
+  b: string,
+  groupOrderArray: string[]
+) {
+  const aIdx = groupOrderArray.findIndex((g) => g === a)
+  const bIdx = groupOrderArray.findIndex((g) => g === b)
+
+  if (aIdx === -1) return 1
+  if (bIdx === -1) return -1
+  if (aIdx > bIdx) return 1
+  if (aIdx < bIdx) return -1
```

Comment:
maybe combine two conditions? ex: `( aIdx === -1 || aIdx > bIdx )`

Reply:
Combined in c241f5c2a0e663a92be6e1e6b1565037ac7572dd.

Reply:
Combined in c241f5c2a0e663a92be6e1e6b1565037ac7572dd.